### PR TITLE
semantic-ui: Increment typescript version to 2.3 to match dependencies

### PR DESCRIPTION
**Pull request by ghost**
This package simply does `/// <reference types` for a bunch of ts2.3 packages, so it should be ts2.3 too.

### CODE REVIEW

**Review by leonard-thieu — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR ready to merge pending express review by a maintainer.

**Pull request merged**
